src/eda_covid19.r

```diff
@@ -0,0 +1,93 @@
+# author: Fatime Selimi, Neel Phaterpekar, Nicholas Wu, Tanmay Sharma
+# date: 2020-11-25
+
+"Creates eda table and plots for the response ratio from the COVID-19 data (from https://github.com/owid/covid-19-data/tree/master/public/data).
+Saves the plots as a pdf and png file.
+Usage: src/eda_covid19.r --input_path=<input_path> --out_dir=<out_dir>
+  
+Options:
+--input_path=<input_path>     Path (including filename) to processed data in csv format
+--out_dir=<out_dir>           Path to directory where the plots should be saved
+" -> doc
+
+library(tidyverse)
+library(knitr)
+library(infer)
+library(cowplot)
+library(docopt)
+library(ggthemes)
+
+opt <- docopt(doc)
+
+main <- function(input_path, out_dir) {
+  
+    #create a table summarizing the key statistics
+    covid_can_usa <- read_csv(input_path)
+    
+    covid_can_usa_summary <-  covid_can_usa %>%
+        group_by(iso_code) %>%
+        summarise(sample_size = n(),
+                  mean_response_ratio = mean(response_ratio),
+                  median_response_ratio = median(response_ratio),
+                  standard_dev = sd(response_ratio)
+        )
+    
+    #create a directory for processed data if it does not exist
+    try({
+      dir.create(out_dir)
+    })
+    
+    #save summary table as a .csv file
+    write_csv(covid_can_usa_summary, 
+              paste0(out_dir, "/covid_can_usa_summary.csv"))
+
```

Comment:
either way works for me, but it seems like it would be easier to have script 3 create the csv and script 4 be the one to create a png in the results folder. That way the script 3 doesn't create/save things into the results folder

Reply:
First, in this script, we're saving the `covid_can_usa_summary.csv` in the results folder which is not quite right. We should save `.csv` files under the `data` and in results we should keep the `.png`s for tables, plots, etc. 
At least this is what we see in Tiffany's example for M1/2.

Second, I think that the summary statistics table is relevant for the EDA but we can create a `.png` for the summary statistics table either in script 3 or script4, should be fine. 

Since you're already creating that `.png` table @nphaterp let's just keep simple and create that `.png` table in script 4. 

P.S. I guess we shouldn't submit the `.csv` files anyway, so I think this PR is good and can be merged to the main branch.